Client/Frontend/Browser/Playlist/Controllers/PlaylistCarplayControllerIOS14.swift

```diff
@@ -0,0 +1,681 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import Combine
+import CarPlay
+import MediaPlayer
+import Data
+import BraveShared
+import Shared
+
+private let log = Logger.browserLogger
+
+@available(iOS 14, *)
+class PlaylistCarplayControllerIOS14: NSObject {
```

Comment:
SonarCloud gonna hate you for it :D
it's ok, we drop iOS13 soon and can address it, maybe create a followup ticket to not forget about it?